kore/src/Kore/Attribute/Pattern.hs

```diff
@@ -156,6 +159,13 @@ isSimplified sideCondition patt@Pattern {simplified} =
     assertSimplifiedConsistency patt
     $ Simplified.isSimplified sideCondition simplified
 
+isSimplifiedIgnoreCondition
```

Comment:
Please add documentation similar to `Kore.Attribute.Pattern.Simplified.isSimplifiedIgnoreCondition`.